src/utils/bytes.js

```diff
@@ -32,3 +32,28 @@ export function stringToBytes(str) {
   }
   return bytes;
 };
+
+/**
+ * Converts a 8-bit bytes array into a string
+ * @param {!Uint8Array} bytes
+ * @return {string}
+ */
+export function bytesToString(bytes) {
+  return String.fromCharCode.apply(String, bytes);
+};
+
+/**
+ * Generate a random bytes array with specific length
+ * @param {!number} length
+ * @return {!Uint8Array}
+ */
+export function getRandomBytesArray(length) {
```

Comment:
OK, cool. The only concern I have is: whether it's time-consuming to use `Math.random` n times? In the share-tracking case, we are only using length of 6, which I guess is fine. @lannka Do you think this is fine?